Android/README.md

```diff
@@ -0,0 +1,3 @@
+# Android Makaleleri
+
+1. [Android Callback](android-callback.md)
```

Comment:
Bunu aslinda `Callback kullanmak suretiyle Android kodumuzda Sepration of Concern Saglamak` seklinde bir baslik ile degistirebiliriz. Bu sayede insanlar konu hakkinda basliga bakarak daha detayli bilgi almis olurlar.

Reply:
Haklısınız hocam, başlık biraz daha geneli kapsıyordu güncelledim.